src/com/company/Main.java

```diff
@@ -8,18 +8,40 @@
 
 
 public class Main {
-    public static void main(String[] args){
+    public static void main(String[] args) {
+
+        String txtFilePath = args[0];
+
+        File file = new File(txtFilePath);
+        if(file.length()==0)
+        {
+            System.out.println("Enter the valid path ");
+            System.exit(0);
+        }
+        isFileFormatSupported(file);
+        isFileExists(file);
+
+        if((isFileExists(file) && isFileFormatSupported(file))== false)
```

Comment:
```suggestion
        if((isFileExists(file) && isFileFormatSupported(file)) == false) {
```
Proper spacing for == and { in the same line

Reply:
I have done that.
ill check for other lines of code if there is any other issue repeated as this.